palaver.cpp

```diff
@@ -190,6 +196,18 @@ class PLVHTTPSocket : public CSocket {
 	CString m_sHostname;
 };
 
+class PLVHTTPNotificationSocket : public PLVHTTPSocket {
```

Comment:
I've made a subclass, so I can attach a token and override `HandleStatusCode` to handle deletion on 401 statuses.